code/obj/machinery/sleeper.dm

```diff
@@ -484,47 +453,57 @@
 			var/oxy = src.occupant.reagents.get_reagent_amount(src.med_oxy)
 			var/tox = src.occupant.reagents.get_reagent_amount(src.med_tox)
 
+			var/injected_anything = FALSE
+
 			// We always inject this, even when emagged to mask the fact we're malfunctioning.
 			// Otherwise, one glance at the control console would be sufficient.
-			if (rejuv < 10)
-				var/inject_r = 5
-				if ((rejuv + 5) > 10)
-					inject_r = max(0, (10 - rejuv))
+			if (rejuv < maximum_reagent)
```

Comment:
```suggestion
			if (rejuv < src.maximum_reagent)
```
(Unsure if `rejuv` should also be `src.rejuv`, as can't see context from here, but easy to check when you're looking at it.)